sympy/sets/ordinals.py

```diff
@@ -50,9 +50,6 @@ def __lt__(self, other):
                 return NotImplemented
         return self._compare_term(other, operator.lt)
 
-    def __hash__(self):
```

Comment:
`hash` should not be removed altogether in a [class that implements `__eq__`](https://docs.python.org/3/reference/datamodel.html#object.__hash__) but it is can be the same as in its parent class:

    If a class that overrides __eq__() needs to retain the implementation of __hash__() from a parent class, the
    interpreter must be told this explicitly by setting __hash__ = <ParentClass>.__hash__.

Here we could have `__hash__ = Basic.__hash__`.